loops-and-arrays.js

```diff
@@ -32,7 +32,7 @@ for(let i = 0; i < names.length; i++) {
 
 console.log('   Opdracht 2');
```

Comment:
Bijna goed. Je hebt de berekening in de console.log geplaatsd waardoor de uitkomst klopt maar de array zelf niet verandert wordt en dat was wel de opdracht. Dus niet:

for(let i = 0; i < numbers.length; i++) {
    console.log(numbers[i] * 3);
}

Maar zo: 

for (let i = 0; i < numbers.length; i++) {
  numbers[i] = numbers[i] * 3;
}

console.log(numbers);